src/components/PlanterDetail.js

```diff
@@ -108,7 +105,14 @@ const PlanterDetail = (props) => {
   return (
     <React.Fragment>
       <Drawer anchor="right" open={props.open} onClose={props.onClose}>
-        <Grid className={classes.root}>
+        <Grid
```

Comment:
Let's make the container and image square to avoid this awkwardness and inconsistency between rotated and unrotated images.

The planter card can also be updated to use a square image.